x-pack/plugins/secops/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -166,7 +166,9 @@ const getAuthenticationColumns = (startDate: number): Array<Columns<Authenticati
     hideForMobile: false,
     render: ({ node }) =>
       has('lastFailure.timestamp', node) ? (
-        <FormattedRelative value={new Date(node.lastFailure!.timestamp!)} />
+        <EuiToolTip position="bottom" content={node.lastFailure!.timestamp!}>
```

Comment:
As we discussed, we should have the same tooltip information for every date in the application.

Reply:
Pointing out you have to be careful with `has` as if `timestamp` is null, has will return a true on you and then you're showing null as content. That might be intended but it has tripped us up before that `has` says if something has a `null` value then it is true.

We might need a has utility at some point soon where if it detects `null` or `undefined` it returns `false`

No changes requested, just pointing out the behavior in case that's not what you wanted. If not, then you need to do:

```ts
has('lastFailure.timestamp', node) && node.lastFailure.timestamp != null ? (
```

Reply:
Thanks for pointing this out! I'll update the conditional to include an additional null check and keep a lookout for other `has`'s with this point in mind. As for the tooltip, this matches the tooltips shown for other `FormattedRelative`'s, so we should be good on this one.